site/content/in-dev/unreleased/configuring-polaris-for-production.md

```diff
@@ -125,6 +125,19 @@ polaris.realm-context.require-header=true
 This will cause Polaris to also return a `404 Not Found` response if the realm header is not present
 in the request.
 
+### Polaris Server Header
```

Comment:
Hm, I think it's better to move this section down to the end of the doc.
It's (at least IMHO) an advanced option for very specific use cases, while the other items mentioned here are all mandatory settings.